src/evcxr.rs

```diff
@@ -1,6 +1,10 @@
 use crate::coord::Shift;
 use crate::drawing::{DrawingArea, IntoDrawingArea};
+use base64;
```

Comment:
As I mentioned previously, I think we should reuse the SVGBackend::blit_bitmap

Though it's a little bit weird that wrap a bitmap in a SVG, but I think this will reduce the number of dependencies we introduced - also reuse the existing code.

Reply:
That's a good idea! I wasn't aware of the SVG base64 implementation, I'll take a look and make the change! I think it's a good practice to reuse existing code.